src/main/drivers/display.c

```diff
@@ -157,26 +155,35 @@ int displayWriteWithAttr(displayPort_t *instance, uint8_t x, uint8_t y, const ch
     return instance->vTable->writeString(instance, x, y, s, attr);
 }
 
-int displayWriteChar(displayPort_t *instance, uint8_t x, uint8_t y, uint8_t c)
+int displayWriteChar(displayPort_t *instance, uint8_t x, uint8_t y, uint16_t c)
```

Comment:
Maybe we should define `displaySymbol_t` type?

Reply:
I think it's not worth the complexity of having to look up what `displaySymbol_t` is. What concerns me about `uint16_t` is that it might make future developers think that any value up to `(2^16) - 1` is accepted. Is there any argument attribute that we can apply to tell the compiler the maximum size (so it will produce a warning when it gets implicitly truncated)?